test/nbrowser/CreateTeamSite.ts

```diff
@@ -0,0 +1,62 @@
+import { assert, driver, Key } from 'mocha-webdriver';
```

Comment:
There's an unfortunate downside with our use of stubs: this test depends on the specific implementation in `CreateTeamModal.ts`, which is what this repo uses, and not what our SaaS repo uses. But our SaaS repo will still try to run this test as-is and will predictably fail.

The `smoke` tests are immune to this, as even our SaaS repo goes through a separate build of grist-core and runs the tests against it.

Let me check with @paulfitz on what we should do here. One idea is to rebrand the `smoke` tests as `core` - for any tests that _must_ be run against a grist-core instance - and move this test there. Another is to add some sort of hook to the test `server` that returns the deployment type (e.g. `core`), and skip this entire test if the deployment type is non-core.

Reply:
In fact it's a problem, waiting for your response

Reply:
Currently tests in `test/nbrowser/*` are assumed to work regardless of what is in stubs, I believe, as George says. How about moving this test to a directory such as `test/nbrowser_with_stubs` or `test/stubs` and updating `package.json` to run tests from there also? Then I think our SaaS will be fine, and (looking ahead) for a repository like `grist-electron` when testing is set up someone could pick and choose whether to run this test based on whether the same team site behavior has been incorporated.

But there's a lot of ways to solve this, so if this suggestion proves complicated we could just pick another :-)

Reply:
It's seems to be a easy fix, I made a commit
Do you think this deserve an explanation somewhere ?